selene_sdk/samplers/multi_sampler.py

```diff
@@ -0,0 +1,324 @@
+"""
+This module provides the `MultiSampler` class, which uses a
+FileSampler for each mode of sampling (train, test, validation).
+The MultiSampler is therefore a subclass of Sampler.
+"""
+import numpy as np
+from torch.utils.data import DataLoader
+
+from .sampler import Sampler
+
+def MultiFileSampler(*args, **kwargs):
+    from warnings import warn
+    warn("MultiFileSampler is deprecated and will be removed from future "
+    "versions of Selene. Please use MultiSampler instead.")
+    return MultiSampler(*args, **kwargs)
+
+class MultiSampler(Sampler):
+    """
+    This sampler draws samples from individual file samplers or data loaders
+    that corresponds to training, validation, and testing (optional) modes.
+    MultiSampler calls on the correct file sampler or data loader to draw
+    samples for a given mode. Example file samplers are under
+    `selene_sdk.samplers.file_samplers` and example data loaders are under
+    `selene_sdk.samplers.dataloaders`.
+
+    MultiSampler can use either file samplers or data loaders for
+    different modes. Using data loaders for some modes while using file samplers
+    for other modes are also allowed. The file samplers parse data files
+    (e.g. bed, mat, or hdf5). The data loaders provide multi-worker iterators
+    that draw samples from online samplers (i.e. on-the-fly sampling). As data
```

Comment:
I changed this - is this correct? only online samplers for data loaders

Reply:
yes that's right - thanks for the edit!